### PR TITLE
Use migration v34

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: ops/generate-daos-subgraph.js [hunk 1]
```diff
@@ -38,6 +38,12 @@ function daoYaml(contract, contractAddress, arcVersion) {
       entities,
       abis: (abis || [contract]).map(contractName => {
         let _arcVersion = Number(arcVersion.slice(arcVersion.length-2,arcVersion.length));
+        if ((_arcVersion >= 34) && (contractName === "UGenericScheme")) {
```
Comment (orenyodfat) [inline ops/generate-daos-subgraph.js]: UGenericScheme is deprecated on 34 . no ?

File: ops/generate-subgraph.js [hunk 2]
```diff
@@ -71,6 +71,12 @@ function combineFragments(fragments, isTemplate, addresses, missingAddresses) {
       abis = (yamlLoad.abis || [contract]).map(contractName => {
         const versionNum = versionToNum(version);
 
+        if ((versionNum >= 34) && (contractName === "UGenericScheme")) {
```
Comment (orenyodfat) [inline ops/generate-subgraph.js]: UGenericScheme is deprecated on 34 . no ?

Comment (ben-kaufman) [reply ops/generate-subgraph.js]: Yes, that's why if a DAO still uses it I have to bring it's ABI from v33

Comment (orenyodfat) [reply ops/generate-subgraph.js]: we should return error in this case

File: ops/mappings.json [hunk 3]
```diff
@@ -527,96 +555,117 @@
...
-        "arcVersion": "0.0.1-rc.33"
+        "arcVersion": "0.0.1-rc.34"
       },
       {
         "name": "DAOTracker",
         "contractName": "DAOTracker",
         "dao": "base",
         "mapping": "DAOTracker",
-        "arcVersion": "0.0.1-rc.33"
+        "arcVersion": "0.0.1-rc.34"
```
Comment (orenyodfat) [inline ops/mappings.json]: lets support 34 also for Rinkeby and main net

**Pull request merged**